tool/cli.js

```diff
@@ -202,8 +203,28 @@ program
   .action(
     tryOrExit(async ({ args }) => {
       const { slug, locale } = args;
-      Document.validate(slug, locale);
-      console.log(chalk.green("✓ All seems fine"));
+      let okay = true;
+      const document = Document.findByURL(buildURL(locale, slug));
+      const { doc } = await buildDocument(document);
+
+      if (doc.flaws) {
+        const flaws = Object.values(doc.flaws)
+          .map((a) => a.length || 0)
+          .reduce((a, b) => a + b);
+        console.log(chalk.red(`Found ${flaws} flaws.`));
+        okay = false;
+      }
+      try {
+        Document.validate(slug, locale);
+      } catch (e) {
+        console.log(chalk.red(e));
+        okay = false;
+      }
+      if (okay) {
+        console.log(chalk.green("✓ All seems fine"));
+      } else {
+        console.log(chalk.red("✗ Found some issues"));
```

Comment:
This feels redundant and a bit confusing to me (e.g., are issues different than flaws?), since before this message is shown, the user will see `Found X flaws.`. For example:`
```sh
rjohnson-44864:deployer rjohnson$(maws_profile)$ yarn tool validate Web
yarn run v1.22.5
$ node tool/cli.js validate Web
Found 4 flaws.
✗ Found some issues
✨  Done in 1.28s.
```
Could we simply skip this message when `okay` is falsy?